botany_importer.py

```diff
@@ -183,49 +185,19 @@ def create_skeleton(self, barcode):
         self.specify_db_connection.commit()
         cursor.close()
 
-
```

Comment:
It seems like the image import method is pretty established now , this has been commented out since I started working here, so at this point I think it should be removed as it won't be used.